app/models/miq_worker.rb

```diff
@@ -336,12 +336,30 @@ def self.close_pg_sockets_inherited_from_parent
     owner_to_pool[Process.ppid].values.compact.each do |pool|
       pool.connections.each do |conn|
         socket = conn.raw_connection.socket
-        _log.info "Closing socket: #{socket}"
+        _log.info("Closing socket: #{socket}")
         IO.for_fd(socket).close
       end
     end
   end
 
+  def queue_name
```

Comment:
Maybe a quick comment for the `queue_name` accessor overrides.  Something like "the `queue_name` might be an array for some workers, so serialize the queue name to JSON"